nemo/collections/nlp/models/token_classification/ner_model.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             config_file=self._cfg.language_model.bert_config,
             checkpoint_file=self._cfg.language_model.bert_checkpoint,
         )
-        self.hidden_size = self.bert_model.config.hidden_size
+        self.hidden_size = self.bert_model.hidden_size
 
```

Comment:
this will break hf bert models. Let's add property hidden_size to the BertModule with default bert_model.config.hidden_size and overwrite it for Megatron

Reply:
That's exactly what we have.

Reply:
@borisfom Megatron has it but BertModule does not

Reply:
You're right, adding.